Package.swift

```diff
@@ -27,7 +27,7 @@ let package = Package(
               .product(name: "FirebaseAnalytics", package: "Firebase"),
             ],
             path: "mParticle-Google-Analytics-Firebase-GA4",
-            exclude: ["Info.plist"],
+            exclude: ["Info.plist", "dummy.swift"],
```

Comment:
Does this address another open PR to delete dummy.swift?

https://github.com/mparticle-integrations/mparticle-apple-integration-google-analytics-firebase-ga4/pull/12